### DISCUSSION TITLE
Why does clickhouse-go client not support bitmap type `AggregateFunction ` like java?

### DISCUSSION THREAD

**Discussion by Cherrison** [ClickHouse/clickhouse-go#1292] [Feature requests]
Why does clickhouse-go client not support bitmap type `AggregateFunction ` like java?
I want to use go to read and write bitmap with clickhouse, but it shows `AggregateFunction` type not support.

**Comment by Cherrison** [comment]
Is there a plan to support this in the future?

**Comment by jkaflik** [comment]
Hi @Cherrison 

We don't have it on our roadmap. It's already register in the issue: https://github.com/ClickHouse/clickhouse-go/issues/1173

Any contributions are welcome.
